users/views.py

```diff
@@ -0,0 +1,84 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponse
+from django.contrib import messages
+from django.contrib.auth import login, authenticate, logout
+from .forms import LoginForm, ResetPasswordForm
+from django.core.mail import send_mail
+from django.utils.http import urlsafe_base64_encode
+from django.contrib.auth.tokens import default_token_generator
+from django.http import BadHeaderError
+from django.utils.encoding import force_bytes
+from django.template.loader import render_to_string
+from django.contrib.auth.models import User
+from django.db.models.query_utils import Q
+
+
```

Comment:
name convention on views methods, like method 'sign_in' is a view so call it 'sign_in_view' , fix this file.